peachjam/views/legislation.py

```diff
@@ -270,6 +270,17 @@ def get_notices(self):
                     }
                 )
 
+        if self.object.date > datetime.now().date():
+            notices.append(
+                {
+                    "type": messages.WARNING,
+                    "html": _(
+                        "This version will be applied on %(current_object_date)s."
```

Comment:
I wouldn't use the word "applied" since that implies an amendment, let's keep it as generic as possible. e.g. "This version's date is in the future, at {date}"? idk